src/Slices/Notification/UI/Drawer/Item.tsx

```diff
@@ -23,18 +22,31 @@ interface Props {
 }
 
 export const Item: React.FC<Props> = ({ notification, onUpdate }) => {
+  const { routeManager } = useContext(DependencyContext);
+  const detailsLink: RouteKindWithId<"CompileDetails"> | undefined =
+    routeManager.getUrlCompileDetailsId(
+      notification.uri
+    ) as RouteKindWithId<"CompileDetails">;
+  const navigate = useNavigateTo();
+
+  const onClick = (notification: Notification): void => {
+    if (!notification.read) onUpdate({ read: true });
+    if (detailsLink) {
+      navigate(detailsLink.kind, { id: detailsLink.params.id });
+    }
+  };
   return (
     <NotificationDrawerListItem
       variant={getSeverityForNotification(notification.severity)}
-      onClick={notification.read ? undefined : () => onUpdate({ read: true })}
+      onClick={() => onClick(notification)}
       isRead={notification.read}
       aria-label="NotificationItem"
     >
       <NotificationDrawerListItemHeader
         variant={getSeverityForNotification(notification.severity)}
         title={notification.title}
       >
-        <ActionList {...{ notification, onUpdate }} />
+        <ActionList {...{ notification, onUpdate, detailsLink }} />
```

Comment:
Is the `detailsLink` required here?

Reply:
no I don't know why I added that, I removed it :p

Reply:
no I don't know why I added that, I removed it :p